jkube-kit/jkube-kit-micronaut/src/test/java/org/eclipse/jkube/micronaut/enricher/MicronautHealthCheckEnricherTest.java

```diff
@@ -34,31 +34,46 @@
 import io.fabric8.kubernetes.api.model.apps.DeploymentBuilder;
 import io.fabric8.kubernetes.api.model.apps.DeploymentSpec;
 import org.assertj.core.api.InstanceOfAssertFactories;
+import org.eclipse.jkube.micronaut.MicronautUtils;
+import org.junit.jupiter.api.AfterEach;
+import org.mockito.MockedStatic;
+import org.mockito.Mockito;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 
 import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.RETURNS_DEEP_STUBS;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
 
 class MicronautHealthCheckEnricherTest {
-
+  private MockedStatic<MicronautUtils> micronautUtils;
   private JKubeEnricherContext context;
   private JavaProject project;
   private KubernetesListBuilder klb;
 
   @BeforeEach
   void setUp() {
-    project = JavaProject.builder()
-        .outputDirectory(new File("target"))
-        .build();
+    context = mock(JKubeEnricherContext.class, RETURNS_DEEP_STUBS);
+    project = mock(JavaProject.class);
```

Comment:
Why did we switch to a Mock instead of reusing the simple POJO?

Reply:
Since there are no mocks, for this issue the only required change should be to remove the dependency from the pom.xml file.

Reply:
@baruKreddy : Looks like I made a mistake while resolving conflicts. Looks like there is no need for mocks in this class.

Could you please do these?

+ update MicronautHealthCheckEnricherTest to [orignal version](https://raw.githubusercontent.com/eclipse/jkube/master/jkube-kit/jkube-kit-micronaut/src/test/java/org/eclipse/jkube/micronaut/enricher/MicronautHealthCheckEnricherTest.java) in master 
+ Remove `src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker`